Pipfile

```diff
@@ -8,7 +8,7 @@ python_version = "3.11"
 
 [packages]
 flask = "*"
-flask-cors = "*"
+flask-cors = "4.0.1"
```

Comment:
Pinned?

Reply:
This has been unpinned. I was running a test and forgot to unpin after.